extensions-core/avro-extensions/src/main/java/org/apache/druid/data/input/avro/AvroFlattenerMaker.java

```diff
@@ -105,7 +104,7 @@ private Object transformValue(final Object field)
       if (binaryAsString) {
         return StringUtils.fromUtf8(((ByteBuffer) field).array());
       } else {
-        return Arrays.toString(((ByteBuffer) field).array());
+        return ((ByteBuffer) field).array();
```

Comment:
a byte buffer is not guaranteed to be array backed, but this is not worse than the prior implementation